style.css

```diff
@@ -7,7 +7,7 @@ Description: WordPress Block theme is a theme that leverages the full-site editi
 Requires at least: 5.9
 Tested up to: 6.0.1
 Requires PHP: 7.4
-Version: 1.0.2
+Version: 1.0.3
 License: Apache License Version 2.0
 License URI: LICENSE
```

Comment:
if there is a license uri, maybe next time we can add this, https://beta-docs.developer.gov.bc.ca/license-your-github-repository/ the License type looks good according to this document.